### PR TITLE
add Pakistan's provinces on registration form of Lums

### PR DESCRIPTION
**Pull request by zubair-arbi**
* Add `lums` feature (`openedx.features.lums`) for custom functionality related to Lums.
* Add lums custom form `UserProfileExtensionForm` to show the Pakistan's provinces dropdown (required) on the registration form of `Lums`
Screen Shot 2019-10-10 at 11 47 54 PM

Screen Shot 2019-10-10 at 11 48 33 PM

Screen Shot 2019-10-11 at 11 32 57 AM

### CODE REVIEW

File: openedx/features/lums/models.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserProfileExtension(models.Model):
+    """
+    This model is used to store additional custom profile fields.
+    """
+
+    empty = 'punjab'
```
Comment (ziafazal) [inline openedx/features/lums/models.py]: nit: I think we should remove this `empty` variable

File: openedx/features/lums/models.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    PROVINCE_CHOICES = (
+        (PUNAJB, _('Punjab')),
+        (SINDH, _('Sindh')),
+        (KPK, _('Khyber Pakhtunkhwa')),
+        (BALOCHISTAN, _('Balochistan')),
+    )
+
+    user = models.OneToOneField(
+        User, unique=True, db_index=True, related_name='profile_extension', on_delete=models.CASCADE
```
Comment (ziafazal) [inline openedx/features/lums/models.py]: nit: we don't need these here `unique=True, db_index=True` django does it by default on `OneToOneField `

Comment (zubair-arbi) [reply openedx/features/lums/models.py]: Done

File: openedx/features/lums/models.py (hunk 1 above)
Comment (zubair-arbi) [reply openedx/features/lums/models.py]: Done

**Pull request merged**